vuepress/docs/next/tutorials/devops/entando-de-app.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebarDepth: 2
+---
+
+# Configure the Entando App Context
+This tutorial describes how to modify the [EntandoApp CRD](../../docs/consume/entandoapp-cr.md) to customize the context path of an Entando App Engine deployment. 
```

Comment:
Minor, CR rather than CRD.

Reply:
"context path" -> "ingress path".

Reply:
@nshaw should any/all of the CRDs on the linked page be corrected?

Reply:
also luigi said "in the entandoapp CRD" ?? i feel like we might need to take another pass at the docs to straighten out CR/CRD mentions ??

Reply:
Yeah, that's incorrect, also incorrect in the opening paragraph of this page. The only CRDs are the cluster-scoped definitions  (all with kind: CustomResourceDefinition) which define what the custom resources (CRs) are.  An EntandoApp is a custom resource. Maybe we can try to fix this page and then open a ticket for further docs review.